pkg/ingester/ingester.go

```diff
@@ -65,6 +67,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.DurationVar(&cfg.MaxChunkIdle, "ingester.chunks-idle-period", 30*time.Minute, "")
 	f.IntVar(&cfg.BlockSize, "ingester.chunks-block-size", 256*1024, "")
 	f.IntVar(&cfg.TargetChunkSize, "ingester.chunk-target-size", 0, "")
+	f.StringVar(&cfg.ChunkEncoding, "ingester.chunk-encoding", chunkenc.EncGZIP.String(), "The algorithm to use for compressing chunk. (gzip, lz4, snappy)")
```

Comment:
this should probably use the range of `supportedEncoding`s `Stringer` methods from `pkg/chunkenc` for the description.